cypress/support/e2e.js

```diff
@@ -92,22 +92,5 @@ Cypress.on("uncaught:exception", (err, _runnable) => {
 });
 
 export const loginHooks = () => {
-  // Global before for logging in to Okta
```

Comment:
the version bump hides a lot of the cookie caching we're doing manually here behind the new `cy.session`